twisted/conch/test/test_keys.py

```diff
@@ -564,6 +564,38 @@ def test_notEqual(self):
         self.assertTrue(rsa1 != None)
 
 
+    def test_fingerprint(self):
```

Comment:
this is for the default format... so this should be named `test_fingerprintDefault` and the docstring should explain the md5 is used when no format is speciffied.

we also need a test to check that when md5-hex format is explicitly call, all works fine

md5hex... so I think that it should be `test_fingerprint_md5_hex()`